src/Symfony/Component/Translation/Tests/DefaultParametersTranslatorTest.php

```diff
@@ -0,0 +1,64 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Translation\Tests;
+
+use PHPUnit\Framework\TestCase;
+use Symfony\Component\Translation\DefaultParametersTranslator;
+use Symfony\Component\Translation\Loader\ArrayLoader;
+use Symfony\Component\Translation\MessageCatalogue;
+use Symfony\Component\Translation\Translator;
+
+class DefaultParametersTranslatorTest extends TestCase
+{
+    public function testTrans()
+    {
+        $translator = new Translator('en');
+        $translator->addLoader('array', new ArrayLoader());
+        $translator->addResource('array', ['welcome' => 'Welcome {name}!'], 'en');
+        $translator->addResource('array', ['welcome' => 'Bienvenue {name}!'], 'fr');
+
+        $globalsTranslator = new DefaultParametersTranslator($translator, ['{name}' => 'Global name']);
+
+        $this->assertSame('Welcome Global name!', $globalsTranslator->trans('welcome'));
+        $this->assertSame('Bienvenue Global name!', $globalsTranslator->trans('welcome', [], null, 'fr'));
+        $this->assertSame('Welcome John!', $globalsTranslator->trans('welcome', ['{name}' => 'John']));
+        $this->assertSame('Bienvenue Jean!', $globalsTranslator->trans('welcome', ['{name}' => 'Jean'], null, 'fr'));
+    }
+
+    public function testTransICU()
+    {
+        if (!class_exists(\MessageFormatter::class)) {
+            $this->markTestSkipped(sprintf('Skipping test as the required "%s" class does not exist. Consider installing the "intl" PHP extension or the "symfony/polyfill-intl-messageformatter" package.', \MessageFormatter::class));
+        }
```

Comment:
```suggestion
    /**
     * @requires extension intl
     */
    public function testTransICU()
    {
```

Reply:
Is this class necessarily loaded? If so, should we will have to update https://github.com/symfony/symfony/blob/4ad506b0855f45061c2d4b2600846a8aaf338f98/src/Symfony/Component/Translation/Tests/TranslatorCacheTest.php#L62 and https://github.com/symfony/symfony/blob/33d71aa7b7c368e9752198413c9fbd0ca8d82617/src/Symfony/Component/Translation/Tests/TranslatorTest.php#L406 too.

Reply:
Yes, this class lives in the intl extension. So, marking the test requiring the extension avoid to check on the class existence on all tests that require intl extension.Am I wrong @xabbuh?

Reply:
Indeed. I updated the tests this way.